src/main/java/uk/gov/hmcts/ccd/domain/service/stdapi/CallbackInvoker.java

```diff
@@ -57,35 +58,45 @@ public void invokeAboutToStartCallback(final CaseEvent caseEvent,
-                                                        final CaseDetails caseDetailsBefore,
-                                                        final CaseDetails caseDetails,
-                                                        final CaseType caseType,
-                                                        final Boolean ignoreWarning) {
+    public AboutToSubmitCallbackResponse invokeAboutToSubmitCallback(final CaseEvent eventTrigger,
+                                                                     final CaseDetails caseDetailsBefore,
+                                                                     final CaseDetails caseDetails,
+                                                                     final CaseType caseType,
+                                                                     final Boolean ignoreWarning) {
+
         final Optional<CallbackResponse> callbackResponse = callbackService.send(
             eventTrigger.getCallBackURLAboutToSubmitEvent(),
             eventTrigger.getRetriesTimeoutURLAboutToSubmitEvent(),
             eventTrigger, caseDetailsBefore, caseDetails);
-        return callbackResponse.flatMap(response -> validateAndSetFromAboutToSubmitCallback(caseType,
-                                                                                            caseDetails,
-                                                                                            ignoreWarning,
-                                                                                            response));
+
+        if (callbackResponse.isPresent()) {
+            return validateAndSetFromAboutToSubmitCallback(caseType,
+                caseDetails,
+                ignoreWarning,
+                callbackResponse.get());
+
+        } else {
+            AboutToSubmitCallbackResponse aboutToSubmitCallbackResponse = new AboutToSubmitCallbackResponse();
+            aboutToSubmitCallbackResponse.setCallBackResponse(Optional.empty());
+            return aboutToSubmitCallbackResponse;
+        }
+
     }
 
     public ResponseEntity<AfterSubmitCallbackResponse> invokeSubmittedCallback(final CaseEvent eventTrigger,
                                                                                final CaseDetails caseDetailsBefore,
                                                                                final CaseDetails caseDetails) {
         return callbackService.send(eventTrigger.getCallBackURLSubmittedEvent(),
-                                    eventTrigger.getRetriesTimeoutURLSubmittedEvent(),
-                                    eventTrigger,
-                                    caseDetailsBefore,
-                                    caseDetails,
-                                    AfterSubmitCallbackResponse.class);
+            eventTrigger.getRetriesTimeoutURLSubmittedEvent(),
+            eventTrigger,
+            caseDetailsBefore,
+            caseDetails,
+            AfterSubmitCallbackResponse.class);
```

Comment:
Previous formatting was correct and shouldn't have been changed.

Reply:
addressed

Reply:
Doesn't look like it has. Configure your formatter to vertically align method parameters when wrapped.